elasticsearch/container.yaml

```diff
@@ -0,0 +1,6 @@
+---
+compose:
+  packages:
+    - elasticsearch
```

Comment:
actually, none of these are needed . . . we don't build elasticsearch from elasticsearch rpm packages anymore, which are essentially wrappers around the maven artifacts, we build it directly from the maven artifacts

Reply:
Hey @richm 
The ART team needs this content so they can consume the corresponding Dockerfiles, and pass them to OSBS to build OCP 3.10. We are having to go back to do this because we need to build x86 and ppc64le containers with ODCS enabled. 
More on the osbs/odcs things:
https://osbs.readthedocs.io/en/latest/users.html#odcs-compose

Once we get this done, we will be able to put the right things in release-3.11 and master branches. 
You don't need these things upstream because OKD doesn't use OSBS to build their containers.